CHANGES.md

```diff
@@ -1,5 +1,30 @@
 # Changes
 
+## edge-21.6.2
+
+This release fixes a problem with the HTTP body buffering that was added
+to support gRPC retries. Now, only requests with a retry configuration
+are buffered (and only when their bodies are less than 64KB).
```

Comment:
`bodies` aren't a quantity comparable to `64kb` you'd need `the size of the bodies` or something.